packages/samples/headless-react/src/pages/DependentFacet.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  buildFacet,
+  buildSearchEngine,
+  Facet as HeadlessFacet,
+  getSampleSearchEngineConfiguration,
+  loadFacetSetActions,
+  SearchEngine,
+} from '@coveo/headless';
+import {Component} from 'react';
+import {Facet} from '../components/facet/facet.fn';
+import {AppContext} from '../context/engine';
+
+export class DependentFacet extends Component<
+  {},
+  {visibleDependantFacet: boolean}
```

Comment:
Suggestion: `showDependentFacet`

Reply:
also careful always writing it Dependent 😄